packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -249,16 +249,16 @@ export default async args => {
             // So we run the same logic for all configs, letting the last one win.
             command([
                 async () => {
+                    if (args.watch) {
```

Comment:
Do we copy now to the deployment folder only if is `npm start`?

Reply:
Yes correct